app/models/spree/calculator/returns/assemblies_default_refund_amount.rb

```diff
@@ -4,7 +4,12 @@ module Spree
   module Calculator::Returns
     class AssembliesDefaultRefundAmount < DefaultRefundAmount
       def compute(return_item)
-        super * return_item.inventory_unit.percentage_of_line_item
+        percentage = return_item.inventory_unit.percentage_of_line_item
```

Comment:
Can we have a whiteline below this? For improved readability :nerd_face:

Reply:
👍